src/observable-array.ts

```diff
@@ -566,15 +566,7 @@ function observableSplice(arr: any[], events: any[], index: number, removeCount:
 	let arr2 = arr as any;
 
 	if (removeCount) {
-		if (removeCount > 1 && arr2.removeRange) {
-			removedItems = arr2.removeRange(index, removeCount);
-		}
-		else if (removeCount === 1 && arr2.remove) {
-			removedItems = [arr2.removeAt(index)];
```

Comment:
This line was throwing an error, since it calls `removeAt` after checking the existence of `remove`.